solt/core/_data.py

```diff
@@ -283,8 +283,8 @@ def to_torch(self, as_dict=False, scale_keypoints=True, normalize=False, mean=No
             elif f == "P":
                 landmarks = torch.from_numpy(el.data).float()
                 if scale_keypoints:
-                    landmarks[:, 0] /= el.width - 1
-                    landmarks[:, 1] /= el.height - 1
+                    landmarks[:, 0] /= el.frame[1] - 1
```

Comment:
Should this already be refactored for nd-compatibility? It seems to be easy to modify...